ios/FluentUI/Controls/IndeterminateProgressBarView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@available(*, deprecated, renamed: "IndeterminateProgressBarView")
+public typealias MSIndeterminateProgressBarView = IndeterminateProgressBarView
+
+/**
+Indeterminate progress bar view
+ */
+@objc(MSFIndeterminateProgressBarView)
+open class IndeterminateProgressBarView: UIView {
+	
+	/// progressbar's track
+	var trackLayer = CALayer()
+
+	/// gradient layer that slides across the track
+	var gradientLayer = CAGradientLayer()
+	
+	@objc override public init(frame: CGRect) {
+		super.init(frame: frame)
+	}
+	
+	required public init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	open override func layoutSubviews() {
+		super.layoutSubviews()
+		
+		trackLayer.frame = CGRect(x: 0, y: 0, width: frame.width, height: frame.height)
+		layer.addSublayer(trackLayer)
+	
+		addGradientLayer()
+		addAnimation()
+	}
+	
+	open override func didMoveToWindow() {
+		super.didMoveToWindow()
+		
+		if let window = window {
+			let lightColor : CGColor = Colors.primary(for: window).withAlphaComponent(0).cgColor
```

Comment:
>withAlphaComponent [](start = 58, length = 18)

please double check with design. are we sure on alpha? we aren't suppoed to use tint or shade variation of primary colors in the fluent palette? alpha is usage should be very minimal because it is against accessibility guidlines.

Reply:
Checked with a designer. Since the edges of the gradient layer are supposed to be transparent (alpha 0) here, it doesn't matter in this case.